### PR TITLE
Feature/soundboard

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
@timple, ready for pull request?

### CODE REVIEW

**Comment by Timple** [comment]
Yup

**Comment by reinzor** [comment]
No clear pr imo, what is the refactor and what is the feature?

**Comment by Timple** [comment]
True, pep8ify should have been seperate pull request

File: scripts/text_to_speech_node.py [hunk 1]
```diff
@@ -38,10 +38,11 @@ def __init__(self, tts_module, executable=None, key=None, pitch=0):
...
 
-        self.character = rospy.get_param("~character",  "Default")
-        self.language  = rospy.get_param("~language",   "us")
-        self.voice     = rospy.get_param("~voice",      "kyle")
-        self.emotion   = rospy.get_param("~emotion",    "Neutral")
+        self.character    = rospy.get_param("~character",    "Default")
+        self.language     = rospy.get_param("~language",     "us")
+        self.voice        = rospy.get_param("~voice",        "kyle")
+        self.emotion      = rospy.get_param("~emotion",      "Neutral")
+        self.samples_path = rospy.get_param("~samples_path", os.path.expanduser("~/MEGA/media/audio/soundboard"))
```
Comment (Rayman) [inline scripts/text_to_speech_node.py]: You should do the `os.path.expanduser` call later so people are able to use `~` in their parameters.

**Review by Rayman — CHANGES_REQUESTED** [review]
Looks good. After the small fix: :shipit:

File: scripts/text_to_speech_node.py (hunk 1 above)
Comment (Timple) [reply scripts/text_to_speech_node.py]: That does make sense!

**Review by Rayman — APPROVED** [review]

**Pull request merged**